### PR TITLE
Override the django Admin Site issues/#42

### PR DESCRIPTION
**Pull request by anshuldutt21**
We needed to override the vanilla admin class. So I did the following tasks-
  -Customized the Admin Site class by creating a class subclassing the Admin Site class and instancing it as shown in the link provided.
  -Registered the  Site, users and organizations and test_project models.
  -Changed the url links accordingly.

### CODE REVIEW

File: openwisp_utils/admin_theme/admin.py [hunk 1]
```diff
@@ -1,22 +1,51 @@
...
+class MyAdminSite(AdminSite):
+     # link to frontend
+    site_url = None
+
+    ## <title>
+    site_title = getattr(settings,
+                          'OPENWISP_ADMIN_SITE_TITLE',
+                          'OpenWISP Admin')
 
-def openwisp_admin(site_url=None):
```
Comment (nemesisdesign) [inline openwisp_utils/admin_theme/admin.py]: this must not be removed in order to retain backward compatibility

File: openwisp_utils/admin_theme/admin.py [hunk 2]
```diff
@@ -1,22 +1,51 @@
+import openwisp_users.models
 from django.conf import settings
 from django.contrib import admin
 from django.utils.translation import ugettext_lazy
+from django.contrib.admin import AdminSite
+from django.contrib.auth import get_user_model
+from openwisp_users.models import (Group, Organization, OrganizationOwner, OrganizationUser,User)
```
Comment (nemesisdesign) [inline openwisp_utils/admin_theme/admin.py]: you must not import test models in the actual library code

File: openwisp_utils/admin_theme/admin.py [hunk 3]
```diff
@@ -1,22 +1,51 @@
...
+admin_site.register(OrganizationOwner)
+admin_site.register(OrganizationUser)
+
+#Models registered for Sites
+admin_site.register(Site)
+
+#Models registered for test_project
+admin_site.register(Operator, OperatorAdmin)
+admin_site.register(RadiusAccounting, RadiusAccountingAdmin)
+admin_site.register(Project, ProjectAdmin)
```
Comment (nemesisdesign) [inline openwisp_utils/admin_theme/admin.py]: you should not do this

File: tests/urls.py [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
+from openwisp_utils.admin_theme.admin import admin_site
 
-openwisp_admin()
 
 urlpatterns = [
     url(r'^accounts/', include('openwisp_users.accounts.urls')),
-    url(r'^admin/', admin.site.urls),
-]
+    url(r'^admin/', admin_site.urls),
+]
```
Comment (nemesisdesign) [inline tests/urls.py]: configure your edito to automatically add new lines at the end of the file

File: openwisp_utils/admin_theme/admin.py (hunk 3 above)
Comment (anshuldutt21) [reply openwisp_utils/admin_theme/admin.py]: I am not able to understand how I should register the models then.  The url admin will just show a blank page.

**Comment by anshuldutt21** [comment]
Hi @nemesisdesign can you review the changes. I have added the admin decorator.

**Comment by nemesisdesign** [comment]
Closed because duplicate of #48

**Pull request closed without merging**